next.config.js

```diff
@@ -1,12 +1,25 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-    reactStrictMode: true,
-    experimental: {
-        scrollRestoration: true,
-    },
-    images: {
-        domains: ["res.cloudinary.com", "images.unsplash.com", "cdn.sanity.io"],
-    }
+  reactStrictMode: true,
+  experimental: {
+    scrollRestoration: true,
+  },
+  images: {
```

Comment:
these should be in a `data` folder with different files such as `footer.ts`

Reply:
moved these into a data folder